src/script/lua_api/l_mainmenu.cpp

```diff
@@ -975,6 +977,49 @@ int ModApiMainMenu::l_gettext(lua_State *L)
 	return 1;
 }
 
+/******************************************************************************/
+int ModApiMainMenu::l_get_secure_settings(lua_State *L)
+{
+	if (ScriptApiSecurity::isSecure(L)) {
```

Comment:
Shouldn't this always be true?
Note: if you did disable backing up globals I think this would return false, maybe change the function to always return true if it's not SSM?